testData/solutions/win32.json5

```diff
@@ -6148,6 +6148,66 @@
             }
         ]
     },
+    "com.microsoft.windows.touchPadSettings": {
+        "name": "Windows precision touchpad settings",
+        "contexts": {
+            "OS": [
+                {
+                    "id": "win32",
+                    "version": ">=5.0"
+                }
+            ]
+        },
+        "settingsHandlers": {
+            "configure": {
+                "type": "gpii.windows.systemSettingsHandler",
+                "capabilities": [],
+                "capabilitiesTransformations": {
+                    "SystemSettings_Input_Touch_SetActivationTimeout": {
+                        "transform": {
+                            "type": "fluid.transforms.valueMapper",
+                            "defaultInputPath": "http://registry\\.gpii\\.net/applications/com\\.microsoft\\.windows\\.touchPadSettings.Sensitivity",
+                            "defaultOutputPath": "value",
+                            "match": {
+                                "low": "Low sensitivity",
+                                "medium": "Medium sensitivity",
+                                "high": "High sensitivity",
+                                "most": "Most sensitive"
+                            },
+                            "noMatch": "Medium sensitivity"
+                        }
+                    }
+                },
+                "supportedSettings": {
+                    "SystemSettings_Input_Touch_SetActivationTimeout": {
+                        "schema": {
+                            "title": "Touchpad sensitivity",
+                            "description": "Changes Windows 10 precision touchpad sensitivity.",
+                            "type": "string",
```

Comment:
Wouldn't we prefer this to be encoded as an enum so that some preferences tool could understand that there is some kind of linear scale?

Reply:
I have removed the current transformation and updated the schema spec as suggested by @the-t-in-rtf .